vue/components/index.js

```diff
@@ -13,5 +13,6 @@ export { thumbnails } from "./thumbnails.vue";
 export { pickers } from "./pickers";
 export { personalization } from "./personalization.vue";
 export { size } from "./size.vue";
+export { loader } from "./loader.vue";

```

Comment:
@gcandal is there a reason for this to not be sorted ?

Reply:
Not besides "it's always been this way". Sorted.